moto/databrew/models.py

```diff
@@ -37,6 +45,20 @@ def create_recipe(self, recipe_name, recipe_description, recipe_steps, tags):
         self.recipes[recipe_name] = recipe
         return recipe
 
+    def update_recipe(self, recipe_name, recipe_description, recipe_steps, tags):
+        if recipe_name not in self.recipes:
+            raise RecipeNotFoundException(recipe_name)
+
+        recipe = self.recipes[recipe_name]
+        if recipe_description:
```

Comment:
What happens if we want to empty the description? I.e., if a user calls:
```
client.update_recipe(.., Description="", ..)
```
For most services this is valid, and means that the description should now be set to an empty string.
So it may be better to explicitly check for a None-value:
```
if recipe_description is not None:
    ...
```

Reply:
Good point, yes will do as you suggested.